projects/docs/docs/manifest/api-reference.md

```diff
@@ -0,0 +1,12 @@
+---
+title: ProjectDescription.framework
+slug: '/manifest/api-reference'
```

Comment:
Should the folder be called `manifests` given we still have different manifests?
```suggestion
slug: '/manifests/api-reference'
```